api/app/controllers/spree/api/v1/checkouts_controller.rb

```diff
@@ -25,6 +25,11 @@ def advance
         end
 
         def update
+          if @order.completed?
```

Comment:
@tommiller1 This check (code) is not needed as ability's order completion check will suffice.